server/routes/nl_interface.py

```diff
@@ -576,6 +573,11 @@ def page():
   if (os.environ.get('FLASK_ENV') == 'production' or
       not current_app.config['NL_MODEL']):
     flask.abort(404)
+  # For the NL module, launch classifier training. This needs to happen here
```

Comment:
Same comment as the one below for nl_interface_next.py

Reply:
why not move this to the nl_server too?

Reply:
Same comment as above